Diffusion/Common/Pipeline/Pipeline.swift

```diff
@@ -50,6 +50,13 @@ class Pipeline {
         return false
     }
 
+    var isSD3: Bool {
+        if #available(macOS 14.0, iOS 17.0, *) {
```

Comment:
Thoughts about making macOS 14 the minimum deployment target? Otherwise, the default model (SD3) would not work on macOS 13.

Reply:
Yes, I agree! I can do it separately after this is merged if you want.

Reply:
I'll add it with this PR now that SD3 is the default anyway.